src/universal/components/MeetingHelp/CheckInHelpMenu.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import {ACTION, CHECKIN, RETROSPECTIVE} from 'universal/utils/constants'
+import withHelpMenu from 'universal/components/MeetingHelp/withHelpMenu'
+import HelpMenuContent from 'universal/components/MeetingHelp/HelpMenuContent'
+import HelpMenuHeader from 'universal/components/MeetingHelp/HelpMenuHeader'
+import HelpMenuBody from 'universal/components/MeetingHelp/HelpMenuBody'
+import {phaseLabelLookup} from 'universal/utils/meetings/lookups'
+import HelpMenuLink from 'universal/components/MeetingHelp/HelpMenuLink'
+
+const linkLookup = {
+  [ACTION]: 'https://www.parabol.co/getting-started-guide/action-meetings-101#social-check-in',
+  [RETROSPECTIVE]:
+    'https://www.parabol.co/getting-started-guide/retrospective-meetings-101#social-check-in'
+}
+
+const CheckInHelpMenu = (props) => {
```

Comment:
broke this into its own component. now its parent doesn't have to worry about its url, content, or header, because the parent shouldn't have to care about that stuff. all it has to do is tell it what kind of meetingType it is.